### PR TITLE
Fix inconsistencies across dropdown elements

### LINKED ISSUE #20475: Fix inconsistencies across dropdown elements
Context: Through community feedback we've heard that the UI of Matomo has room for improvement. We are therefore looking at improvements (small and large) that can have an impact on making the platform easier to use, particularly for new users who are struggling the most.

Problem: Dropdown elements have visual inconsistencies, leading to an unnecessarily difficult UX 

Proposed solution:
- move icons on dropdowns to after the text
- make the icons consistent (e.g. the standard v icon pointing downwards when folded in, and pointing up when folding out)
- check that these controllers are consistent  in the whole codebase (core + plugins)

Current
image

New proposal
image

### PR DESCRIPTION
**Pull request by bx80**
### Description:

Move the main menu dropdown icons to the right.

Affects: 
- Site selector
- Period selector
- Segment selector
- Dashboard selector
- Tag Container selector
- Ab test period selector
- Version info header
- Quick access

Fixes #20475

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: tests/UI/expected-screenshots/UIIntegrationTest_dashboard5_mobile.png
Comment (sgiehl) [inline tests/UI/expected-screenshots/UIIntegrationTest_dashboard5_mobile.png:1]: @Javi-Ormaechea Actually not relevant for the changes around drop-downs, but do you think that it would look better if the icon in the search input would also be placed right? Especially in this mobile view it looks a bit weird that only the one icon is placed left...

File: plugins/Dashboard/tests/UI/expected-screenshots/DashboardManager_create_new.png
Comment (sgiehl) [inline plugins/Dashboard/tests/UI/expected-screenshots/DashboardManager_create_new.png:1]: @bx80 Would you mind improving the spacing of dropdown arrow in the main menu? The group selection drop down for dashboards doesn't look fine (might be the same for goals as well). Already saw that when the icons were replaced, but thought this might get changed with drop downs anyway, so didn't mention it there...

**Review by sgiehl — CHANGES_REQUESTED** [review]
Left one relevant comment for this PR, that would be good to changed. Otherwise it's looking fine so far

File: plugins/Dashboard/tests/UI/expected-screenshots/DashboardManager_create_new.png
Comment (bx80) [reply plugins/Dashboard/tests/UI/expected-screenshots/DashboardManager_create_new.png:1]: Good spot :+1:
 I've adding some extra padding.
image

File: tests/UI/expected-screenshots/UIIntegrationTest_dashboard5_mobile.png
Comment (bx80) [reply tests/UI/expected-screenshots/UIIntegrationTest_dashboard5_mobile.png:1]: I've made this change because the mobile search input now looks awful on the left, here it is after the change.
image

Comment (bx80) [reply tests/UI/expected-screenshots/UIIntegrationTest_dashboard5_mobile.png:1]: and before:
image

Comment (Javi-Ormaechea) [reply tests/UI/expected-screenshots/UIIntegrationTest_dashboard5_mobile.png:1]: I agree that it doesn't look great on this specific screen but it should look consistent on desktop too. If we make this change on mobile it should also happen on desktop. If we do that, then desktop is inconsistent. 

In my opinion what we need to optimise here are these filters for mobile (on a new ticket) and not focus on the search icon's placement as it should be something we look at across all breakpoints and components to have consistency (on a new ticket - icon placement).

For now happy with this change.

Comment (bx80) [reply tests/UI/expected-screenshots/UIIntegrationTest_dashboard5_mobile.png:1]: Sounds good, if we have some sort of standard across both mobile and desktop then it'll make these sort of changes easier :+1:

**Review by sgiehl — APPROVED** [review]
Changes are looking good now. Merged in the latest changes from 5.x-dev and will wait for the tests to finish before merging...

**Comment by michalkleiner** [comment]
Apologies for my sometimes annoying sense for tiny details. I can't help it but I think the new chevron icon for the dropdown is too thick for where/how it's used. I reckon we should explore the option of either making it slightly smaller or perhaps lighter, and also push it vertically down a pixel or two to be better aligned on the row, both in the dropdowns and the lefthand menu.

**Comment by michalkleiner** [comment]
Just compare the line thickness and how it sort of stands out for the chevron compared to the other two icons to the left
Screenshot 2023-08-30 at 5 45 19 PM

**Comment by sgiehl** [comment]
@michalkleiner I agree that the icon could maybe be placed slightly better. But that was actually already the case before, as the icons were only moved to the other side. Maybe @Javi-Ormaechea should have a look and give his opinion. 
I will merge this one now anyway, as it won't change anything in terms of effort if it would be added here or changed in a new PR.

**Pull request merged**
